beacon_node/beacon_chain/src/builder.rs

```diff
@@ -80,7 +80,6 @@ pub struct BeaconChainBuilder<T: BeaconChainTypes> {
     slot_clock: Option<T::SlotClock>,
     shutdown_sender: Option<Sender<&'static str>>,
     head_tracker: Option<HeadTracker>,
-    data_dir: Option<PathBuf>,
```

Comment:
totally in favour of axing this, I noticed that there are a few weird redundant things in here when working on #2234